Calculator.php

```diff
@@ -8,32 +8,19 @@
 const AGREE = 'yes';
 const DEGREE = 'no';
 const HISTORY = 'history';
+const SYSTEM_COMMANDS = [QUIT, INFO, HISTORY];
+
 const AVAILABLE_COMMANDS = ['+', '-', '*', '/', '^', 'sr', QUIT, INFO, HISTORY];
 
 while ($times < 1) {
     info('Enter ' . INFO . ' to see available commands.');
 
     $command = choose('Enter command: ', AVAILABLE_COMMANDS);
 
-    if ($command == INFO) {
-        info((implode(' ;  ', AVAILABLE_COMMANDS)) . ' ;');
-
-        continue;
-    }
-
-    if ($command == QUIT) {
-        $command = choose('Are you sure to wanna quit? Yes/No ', [AGREE, DEGREE]);
-
-        if ($command == AGREE) {
-            exit();
-        }
-
-        continue;
-    }
+    $isSystemCommand = in_array($command, SYSTEM_COMMANDS);
 
-    if ($command == HISTORY) {
-        $history = file_get_contents('history.txt');
-        info($history);
+    if ($isSystemCommand) {
```

Comment:
we can use condition directly without the separate variable

```suggestion
    if (in_array($command, SYSTEM_COMMANDS)) {
```